cli.py

```diff
@@ -32,8 +32,73 @@
 @click.argument("serve", required=True)
 def start(adduser, api, auth, dburl, hydradoc, port, serverurl, token, serve):
     """Python Hydrus CLI"""
```

Comment:
This docstring is not informative. The parameters description is missing.
Every method should be documented as:
```
def test_method(arg1, arg2, ...):
    """
    Description of what the method performs.

    Generic Notes to consider when running the method if any

   : param arg1: what kind of value is expected
   : param arg2: what kind of value is expected
   :return : What is returned or `None`
"""
```

Also add a general documentation for CLI usage and features at https://github.com/HTTP-APIs/hydra-ecosystem-wiki/issues/10